android/app/build.gradle

```diff
@@ -119,8 +119,7 @@ android {
 }
 
 project.ext.vectoricons = [
-        // Intentionally left blank
```

Comment:
Do we know why it was left blank on purpose?

Reply:
Reverted